### DISCUSSION TITLE
Is there a list of sorting keywords

### DISCUSSION THREAD

**Discussion by johnwoltman** [fooyin/fooyin#190] [Q&A] [answered]
Hi, I'd like to be able to sort my collection a few different ways and I found the Sort Script settings but I don't know where all the different keywords (variables?) are.  For instance, I'd like to be able have it sorted first by Library Path or Library Name.  Other times I'd like just Album Artist.  Is there a list somewhere?

**Comment by ludouzi** [comment, marked as answer]
There will be a detailed list soon as part of the documentation I'm currently writing, but for now here's a list of variables specific to tracks:

`%title%` - Falls back to filename if empty
`%artist%` - Falls back to the following fields if empty: album artist, composer, performer
`%uniqueartist%` - Track artists not in album artists
`%album%`
`%albumartist%` - Falls back to the following fields if empty: artist, composer, performer
`%track%`
`%tracktotal%`
`%disc%`
`%disctotal%`
`%genre%`
`%composer%`
`%performer%`
`%duration%` - Milliseconds
`%duration_s%` - Seconds
`%lyrics%`
`%comment%`
`%date%`
`%year%`
`%filesize%`
`%bitrate%`
`%samplerate%`
`%firstplayed%`
`%lastplayed%`
`%playcount%`
`%rating%` - 0 to 5
`%codec%`
`%channels%`
`%bitdepth%`
`%addedtime%`
`%lastmodified%`
`%filepath%`
`%relativepath%` - File path relative to library
`%filename%` - Without extension
`%extension%`
`%filename_ext%` - With extension
`%path%` - Path to containing directory
`%libraryname%`
`%librarypath%`

**Reply by johnwoltman** [reply]
Thank you very much!
